nucypher/control/controllers.py

```diff
@@ -310,14 +310,25 @@ def handle_request(self, method_name, control_request, *args, **kwargs) -> Respo
                            self.emitter.MethodNotFound)
 
         try:
+            # handle request headers
+            request_headers = control_request.headers
+            request_headers = {header.upper(): value for (header, value) in request_headers.items()}
+
             request_data = control_request.data
-            request_body = json.loads(request_data) if request_data else dict()
+            # handle bytes in request body
+            if request_headers.get('CONTENT-TYPE') == 'application/octet-stream':
+                request_body = {"data": bytes(request_data)}
```

Comment:
If the request header specifies a content type of bytes, isn't the body already bytes?  Is this just a matter of getting the bytes into a dictionary for consumption?